autopilot/model-explainability/explaining_customer_churn_model.ipynb

```diff
@@ -90,7 +90,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!python -m pip install shap"
+    "conda install -c conda-forge shap"
```

Comment:
Missing `!` prior to running system call

Reply:
`%conda install` will use the appropriate flags for a better installation of conda packages.